src/lib/awsIamRole.js

```diff
@@ -38,25 +39,33 @@ var deleteIAMRole = function(iam, options, cb) {
 }
 */
 var createRole = function(iam, options, cb) {
-  var extid = options.externalId;
-  options.assumeRolePolicyDocument.Statement[0].Condition.StringEquals = {"sts:ExternalId": extid};
-  var params = {
+   var extid = options.externalId;
+   const awsroles = JSON.parse(fs.readFileSync('./json/federate-role_info.json', 'utf8'));
```

Comment:
Do we need this? We have existing functionality to create IAM roles and attach inline policy stored in federate-role_info.json.

Reply:
Please review the latest changes.